andela/titan-dev-ah-frontend
# 164215001 fix navigation bar on home page

#### What does this PR do?
Fix the navigation bar issue on the homepage
#### Description of Task to be completed?
- add the new version of the navigation bar on the new-article page
#### How should this be manually tested?
- Fetch origin
- Check out the branch bg-fix-navigation-bar-#164215001
- Open the following file pages/new-article.html in chrome or firefox
- And use the inspector to view  with different devices.
#### Any background context you want to provide?
- After merging ft-create-article-interface-#164215001 the nav bar was messed up
#### What are the relevant pivotal tracker stories?
#164215001
#### Screenshots (if appropriate)
<img width="1265" alt="Screen Shot 2019-03-15 at 18 15 59" src="https://user-images.githubusercontent.com/18573157/54445901-6e983200-474e-11e9-984e-059981aaa599.png">

#### Questions:
